### PR TITLE
Update deprecated reference

### PR DESCRIPTION
**Pull request by aeng**
Update all deprecated reference.

### CODE REVIEW

File: zanata-common-api/src/main/java/org/zanata/rest/GenerateSamples.java [hunk 1]
```diff
@@ -40,7 +40,7 @@
 
     public GenerateSamples(PrintStream out) {
         mapper = new ObjectMapper();
-        mapper.getSerializationConfig().enable(Feature.INDENT_OUTPUT);
+        mapper.getSerializationConfig().with(Feature.INDENT_OUTPUT);
```
Comment (seanf) [inline zanata-common-api/src/main/java/org/zanata/rest/GenerateSamples.java]: This whole class should either be reworked into a unit test, documented as a utility, or else deleted before it costs any more maintenance.  I think Asgeir wrote it, so we'll have to work out whether it has any use just from reading the code.

https://code.google.com/p/flies/source/list?path=/common/zanata-common-api/src/main/java/org/zanata/rest/GenerateSamples.java&r=de6d7ad81b8c051882f9e1586f70174459418ec5

**Comment by huangp** [comment]
:+1:

**Comment by aeng** [comment]
Merged into integration/master. https://github.com/zanata/zanata-api/commit/6d37c666a849cb5a96dd9e926a7290ba4ed297cc

File: zanata-common-api/src/main/java/org/zanata/rest/client/IFileResource.java [hunk 2]
```diff
@@ -51,10 +51,15 @@
 @Consumes({ MediaType.APPLICATION_OCTET_STREAM })
 public interface IFileResource extends FileResource {
 
+    /**
+     * Deprecated.
+     * @see #acceptedFileTypeList
```
Comment (seanf) [inline zanata-common-api/src/main/java/org/zanata/rest/client/IFileResource.java]: This should really be part of a `@deprecated` tag in the javadoc.

**Pull request closed without merging**